libqtile/core/manager.py

```diff
@@ -775,7 +775,11 @@ def get_interesting_pid(win):
             # managed and don't have any state.
             if not isinstance(win, window.Window):
                 return None
-            return win.window.get_net_wm_pid()
+            try:
+                return win.window.get_net_wm_pid()
+            except Exception:
+                # the X connection may be invalid due to server crash
```

Comment:
Can we add some logging here? It'd be good to see if this is something that is triggering for other reasons.

Reply:
Right, that's better, I've replaced the comment line with a logging.